src/main/java/org/jetbrains/bsp/bazel/server/ParsingUtils.java

```diff
@@ -0,0 +1,125 @@
+package org.jetbrains.bsp.bazel.server;
+
+import ch.epfl.scala.bsp4j.Diagnostic;
+import ch.epfl.scala.bsp4j.DiagnosticSeverity;
+import ch.epfl.scala.bsp4j.Position;
+import ch.epfl.scala.bsp4j.Range;
+import com.google.common.base.Splitter;
+import com.google.common.io.Files;
+import com.google.devtools.build.lib.buildeventstream.BuildEventStreamProtos.Progress;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jetbrains.bsp.bazel.common.Constants;
+
+public class ParsingUtils {
+
+  private static final Logger LOGGER = LogManager.getLogger(ParsingUtils.class);
+
+  private static final String ERROR = "ERROR";
+  private static final String ERROR_PREAMBLE = ERROR + ": ";
+  private static final String AT_PATH = " at /";
+
+  public static URI parseUri(String uri) {
+    try {
+      return new URI(uri);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  // TODO FOR REVIEW: naming suggestions? what exactly is the thing that it returns?
+  public static List<String> parseClasspathFromFile(URI protoPathUri) {
```

Comment:
This is obtaining directly the jars location from a bazel aspect. I would suggest calling the method `parseClasspathFromAspect` and the variable `dependenciesAspectOutput` (URI is already outlined by the type)